doomrainbow/src/main/java/com/detroitlabs/kyleofori/doomrainbow/AngleUtils.java

```diff
@@ -7,12 +7,35 @@ public static float getRadiusCosineCoefficient(final float valuePositionInDegree
         return (float) Math.cos(valuePositionInRadians);
     }
 
-    public static float convertFromValueToAngle(
+    public static float getInteriorAngleFromValue(
             final float value,
-            final float distanceBetweenExtremeAngles,
-            final float distanceBetweenExtremeValues) {
+            final float exteriorStartValue,
+            final float exteriorEndValue,
+            final float exteriorStartAngle,
+            final float exteriorSweepAngle) {
+
+        if (exteriorStartValue < exteriorEndValue) {
+            if(value <= exteriorStartValue) {
+                return exteriorStartAngle;
+            } else if (value >= exteriorEndValue) {
+                return exteriorStartAngle + exteriorSweepAngle;
+            } else {
```

Comment:
As Stu once told me, [don't repeat yourself](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself). 
The code in this else block could be made into its own method so it doesn't have to be repeated.